src/apiflask/scaffold.py

```diff
@@ -55,9 +55,14 @@ def handle_error(  # type: ignore
             error_headers,
         )
 
+    def load_location_data(self, schema: Schema, location: str) -> t.Any:
+        """
+        Expose the internal `_load_location_data` method to support loading data without validation
+        """
+        return self._load_location_data(schema=schema, req=flask_request, location=location)
+
 
 parser: FlaskParser = FlaskParser()
-use_args: t.Callable = parser.use_args
```

Comment:
The only reference to this function was within the `@input` decorator call to `use_args`, and I find it a bit more obvious to just reference the singleton `parser` instance directly, so I decided to remove this variable.